virtualmachine/virtualmachine.go

```diff
@@ -22,6 +22,8 @@ type VirtualMachine interface {
 	Halt() error
 	Start() error
 	GetSSH(ssh.Options) (ssh.Client, error)
+	AddDisk() error
+	RemoveDisk() error
```

Comment:
This need not be merged with apcera/libretto. For now as we are supporting vsphere so that wouldn't hamper the virtualmachine-vsphere connection. As soon as other clouds come in we will implement these functions for other clouds as well.